backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -235,6 +243,22 @@ export const createUpdateCampDtoValidator = async (
   return next();
 };
 
+export const createCampDtoValidator = async (
```

Comment:
These were the two new validators created (create, update), basically just calling the same validator as before but now passing in the type.